### DISCUSSION TITLE
Examples of using the @nuxtjs/toast or any vue toast plugin?

### DISCUSSION THREAD

**Discussion by procrates** [nuxt/nuxt#18028] [Questions] [answered]
I tried adding different vue 3 toast plugins to nuxt using the vue plugins docs. No success.
Also tried adding nuxt toast, also no success.

Anyone has any examples please? Thanks!

**Comment by DOOM84** [comment, marked as answer]
I am using it like this. in plugins/showToast.client.js :

```javascript
import {defineNuxtPlugin} from '#app'
import Toast, {POSITION, useToast} from "vue-toastification";
import "vue-toastification/dist/index.css";

const toast = useToast();

export default defineNuxtPlugin(nuxtApp => {
    nuxtApp.vueApp.use(Toast, {
        position: POSITION.TOP_RIGHT,
    });
    nuxtApp.provide('showToast', (msg, type, time = 5000) => toast[type](msg, {timeout: time}))
})
```

in template on client: 
```javascript
const {$showToast} = useNuxtApp();
$showToast('Hello world', 'info', 2000);
```

**Reply by procrates** [reply]
Thank you very much doom! All of those small things makes it much easier to work with new framworks!

I guess this is the way to implement other vue plugins too?
The docs shows how g-tag works, but not how it is implemented or how to access its properties in ex. Page setup.

**Reply by DOOM84** [reply]
Yes. I've implemented in this way the several other plugins like vue-i18n, vue-scrollto, moment.js.
